deploy/operator.yaml

```diff
@@ -79,6 +79,8 @@ spec:
               value: quay.io/che-incubator/devworkspace-che-operator:ci
             - name: RELATED_IMAGE_devworkspace_controller
               value: quay.io/devfile/devworkspace-controller:next
+            - name: RELATED_IMAGE_internal_rest_backup_server
+              value: quay.io/eclipse/che-backup-server-rest:eeacd92
```

Comment:
We have to contact Nick how to productize this image.

Reply:
I've created [UBI based dockerfile](https://github.com/che-dockerfiles/che-backup-server-rest) too. @nickboldt please let me know if any other steps are needed from my side.

Reply:
lots of steps required to productize a new container. See https://issues.redhat.com/browse/CRW-1878